src/components/coin.ts

```diff
@@ -81,6 +81,14 @@ export const getCoinBalanceByUserId = async (userId: string): Promise<number> =>
   return _.get(res, 'balance', 0);
 };
 
+export const getPrivateUserIdList = async (userId: string): Promise<number> => {
+  const db = await openDB();
+  // Query user privacy from DB.
+  const res = await db.get('SELECT is_private FROM user_coin WHERE user_id = ?', userId);
```

Comment:
corresponding update statement for how ppl can set that they wanna be private?
also might need to change DB schema code for new column

Reply:
Hope I got everything...